src/components/button/templates/component/template.js

```diff
@@ -92,8 +96,22 @@ export function componentTemplate({ props } : { props : Object }) : string {
         throw new Error(`Unexpected size for ${label} button: ${size}`);
     }
 
-    let logoColor = conf.logos ? conf.logos[color] : '';
+    let logoColor;
+
+    // button config override for branded buy now button
+    if (props.style.branding && label === 'buynow') {
+        contentText = `\$\{pp\}\$\{paypal\} ${contentText}`;
+        conf.tagline = true;
```

Comment:
We shouldn't mutate the conf object as this will potentially affect other buttons. Can we use a local variable instead?